testproject/Assets/Tests/Manual/Scripts/ManualNetworkVariableTest.cs

```diff
@@ -163,11 +128,6 @@ private void ValueChanged(int before, int after)
                     m_Problems += "Didn't receive any NetworkDictionary updates from other machines";
                 }
 
-                if (Math.Abs(m_MaxDelta - m_MinDelta) > 5)
-                {
-                    m_Problems += "Delta range: " + m_MinDelta + " + " + m_MaxDelta + "\n";
-                }
-
                 if (m_Problems == "")
                 {
                     Debug.Log("**** TEST PASSED ****");
```

Comment:
I think this the only test where we dump TEST PASSED / TEST FAILED to the screen; I'd delete for consistency